### ISSUE TITLE
Android gradle version update needed

### ISSUE THREAD

**Issue by aman0408** [debutdeveloper/sms_retriever#11]
FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.3.10 and higher.
The following dependencies do not satisfy the required version:
project ':sms_retriever' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.71

**Comment by AsadSaleh** [comment]
yeah, we need to do something. Anyone understand and can do a PR?

**Comment by britolucaspatrick** [comment]
Estou tendo o mesmo problema.

**Comment by muriloe** [comment]
Same problem here, there some easy fix?

**Comment by thebinarysolution** [comment]
please do a pull request and fix those versions, ill merge it. Thanks

**Comment by AliEasy** [comment]
same problem

**Comment by britolucaspatrick** [comment]
> same problem

https://pub.dev/packages/smsretrieverptk
